prql-compiler/prqlc/Cargo.toml

```diff
@@ -18,9 +18,12 @@ color-eyre = "0.6.1"
 env_logger = {version = "0.9.1", features = ["termcolor"]}
 itertools = "0.10.3"
 notify = "^5.1.0"
-prql-compiler = {path = '..', version = "0.5.2" }
+minijinja = {git = "https://github.com/aljazerzen/minijinja"}
```

Comment:
@aljazerzen we should release this as soon as possible; as at the moment we can't release PRQL...